__tests__/index.test.ts

```diff
@@ -1,8 +1,8 @@
 // @flow
 
-import nock from 'nock'
+import * as nock from 'nock'
 
-import {autocomplete, bulk, reverse, search} from '../index'
+import { autocomplete, bulk, reverse, search } from '../dist/index'
```

Comment:
I don't fully get why we import from dist here. Could a comment help explain this?

Reply:
Good call. Comment added in 503172a